src/main/java/com/hubspot/jinjava/el/ext/ExtendedParser.java

```diff
@@ -578,4 +628,20 @@ public AstNode createAstNode(AstNode... children) {
       return null;
     }
   };
+
+  private static boolean shouldUseNaturalOperatorPrecedence() {
+    return (
+      JinjavaInterpreter.getCurrent() != null &&
+      JinjavaInterpreter
+        .getCurrent()
+        .getConfig()
+        .getLegacyOverrides()
+        .isUseNaturalOperatorPrecedence()
+    );
```

Comment:
Theres
`JinjavaInterpreter.getCurrentMaybe().map(interpreter -> interpreter.getConfig().getLegacyOverrides().isUseNaturalOperatorPrecedence()).orElse(false)` too if you're big on writing expressions like that

Reply:
Ooh I very much am, thanks! 😄 I don't know why I didn't think to just to an `Optional.ofNullable` myself.